### DISCUSSION TITLE
自定义边，upsert HTML 类型节点时候，提示 this.context 是个 undefined

### DISCUSSION THREAD

**Discussion by Juinvok** [antvis/G6#6672] [Q&A]
![image](https://github.com/user-attachments/assets/5d2e15e7-7285-4060-9d83-c281b583632c)
自定义边，upsert HTML 类型节点时候，提示 this.context 是个 undefined

**Comment by petercat-assistant** [comment]
在 antvis/G6 的问题库中没有找到与 "this.context 是个 undefined" 相关的已知问题。这个问题可能是由于在自定义边或节点时，某些上下文对象未正确初始化或传递导致的。

为了帮助您解决这个问题，您可以检查以下几点：

1. **确保上下文初始化**：在创建自定义边或节点时，确保所有需要的上下文对象都已正确初始化。

2. **检查节点和边的配置**：查看是否在配置中遗漏了某些必需的属性或方法。

3. **调试代码**：使用调试工具查看在 upsert 操作时，`this.context` 的状态，找出是在哪一步变成了 `undefined`。

如果您有更多的代码片段或上下文信息，我可以帮助您更具体地分析问题。

如果需要进一步的帮助，请随时告诉我！ 

<details>
<summary>🪧 Tips</summary>
For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.
</details>

**Comment by liaodalin19903** [comment]
可以贴一下你的全部代码细节。
